api/src/wfl/api/handlers.clj

```diff
@@ -43,6 +43,12 @@
           query {:includeSubworkflows false :start start :end end}]
       (succeed {:results (cromwell/query env query)}))))
 
+(defn strip-internals
+  "Strip internal properties from the `workload` and its `workflows`."
+  [workload]
+  (let [prune #(apply dissoc % [:id :items])]
+    (prune (update workload :workflows (partial mapv prune)))))
```

Comment:
It took me some time to understand what this is doing, very elegant and extensible!

(I wish Postgres had something like BigQuery's `SELECT * EXCEPT(ColumnNameX, [ColumnNameY, ...])`)

Reply:
The thing is, we want this stuff for the innards of wfl (private state). We just don't want them in the workload responses.